dev/com.ibm.ws.beanvalidation.v20.cdi/src/com/ibm/ws/beanvalidation/v20/cdi/internal/ValidationReleasableFactoryImpl.java

```diff
@@ -45,7 +53,7 @@
 /**
  * An implementation that is CDI aware.
  */
-@Component(configurationPolicy = ConfigurationPolicy.IGNORE,
+@Component(configurationPolicy = ConfigurationPolicy.OPTIONAL,
            immediate = true,
            property = { "type=CDIValidationReleasableFactory" })
```

Comment:
this property can be removed if we're going to define it in metatype/config anyway

Reply:
Creating follow-up issue for this.